### PR TITLE
Validate building form inputs

### PR DESCRIPTION
**Pull request by JosMigue**
@TheUrbina20 Can you please review this code

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.contenedor{
```
Comment (TheUrbina20) [inline public/css/style.css]: This is very similar to `container`, if you want to override the default style of that class, pay attention on the css specificity.

File: database/migrations/2020_03_31_000810_create_cfdis_table.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('cfdis', function (Blueprint $table) {
+            $table->id();
+            $table->string('c_cfdi')->unique();
+            $table->string('namecfdi');
+            $table->boolean('pFisica');
```
[the remark this answers is not in the document]
Comment (JosMigue) [reply database/migrations/2020_03_31_000810_create_cfdis_table.php]: Got it, the cfdis can be moral and físical, not just one

File: public/js/forms.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    if(isInputUserNumber){
+      document.getElementById('messageNumber').style.display='none';
+    }else{
+      document.getElementById('messageNumber').style.display='block';
+    }
+  }
+}
+
+function checkIfItsEmpty(inputValue){
+  return inputValue === '' ?  true : false;
```
Comment (TheUrbina20) [inline public/js/forms.js:32]: Useless function. Avoid one line function

File: public/js/forms.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+  function checkFormatRfc(input){
+    const rfcValue = input.value;
+    const rfcFormat = new RegExp("^([A-ZÑ\x26]{3,4}([0-9]{2})(0[1-9]|1[0-2])(0[1-9]|1[0-9]|2[0-9]|3[0-1])([A-Z]|[0-9]){2}([A]|[0-9]){1})?$");
+    const isUserInputCorrectRfc = rfcFormat.test(rfcValue);
+    if(checkIfItsEmpty(rfcValue)){
```
Comment (TheUrbina20) [inline public/js/forms.js:5]: Pls move all this login to something like `toggleMessageContainer(elementId, visibility)`

**Pull request merged**